rep-2000.rst

```diff
@@ -547,6 +547,124 @@ Build System Support:
+|  rmw_fastrtps_cpp        | eProsima Fast-RTPS  | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+|  rmw_connext_cpp         | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
+|                          |                     |               | and OpenEmbedded            |                                      |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_opensplice_cpp       | ADLINK OpenSplice   | Tier 2        | All Platforms except Debian | All Architectures                    |
+|                          |                     |               | and OpenEmbedded            |                                      |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.7
+
+
+Dependency Requirements:
+
++-------------+-------------------------------------------------+------------------------------------+
+|             |                        Required Support         |        Recommended Support         |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Package     |  Ubuntu Focal  |     MacOS**   |   Windows 10** | Debian Buster  | OpenEmbedded Thud |
++=============+================+===============+================+================+===================+
+| CMake       |     3.15.4     |     3.14.4    |     3.14.4     |      3.13.4    |       3.12.2      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| EmPY        |                                      3.3.2                                           |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Gazebo      |     9.0.0      |     9.9.0     |      N/A       |      9.8.0*    |        N/A        |
```

Comment:
We're planning to upstream Gazebo 11 to Ubuntu Focal and Debian. The latest Gazebo 11 version will also be available through Homebrew for Mac.

On the Ignition side, we'll have Citadel on Ubuntu / Debian / Homebrew.

Reply:
Updated Gazeob (1467522) and added Ignition (ee62dc3)

Reply:
@chapulina It is my understanding that we didn't get things upstreamed in time. Can you confirm if the Gazebo/Ignition versions for Foxy need to be changed or if it's okay to leave as-is?

Reply:
As discussed offline, we'll be releasing Gazebo 11 and Ignition Citadel into http://packages.ros.org to make it easier for ROS users to use them.

Reply:
@chapulina Please update the corresponding lines in the table then.